specification/sections/01-flag-evaluation.md

```diff
@@ -426,7 +426,8 @@ stateDiagram-v2
     STALE --> NOT_READY:shutdown()
     ERROR --> NOT_READY:shutdown()
     READY --> RECONCILING:::client:setContext()
-    RECONCILING:::client --> READY:*
+    RECONCILING:::client --> READY
```

Comment:
Did you intend to remove the asterisk?

Reply:
Yes this is not emitted spontaneously from the provider, it's part of the lifecycle (triggered by SDK events).